src/config/main.ts

```diff
@@ -14,19 +14,30 @@
  * limitations under the License.
  */
 import Graph from './../component/graph/Graph';
+import { DropFileUserInterface } from '../component/ui_ux/DropFileUserInterface';
 
 export const graph = new Graph(window.document.getElementById('graph'));
+const INITIALIZE_DRAG_AND_DROP = true;
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-function handleFileSelect(evt: any): void {
-  const f = evt.target.files[0];
-
+// callback function for opening | dropping the file to be loaded
+function readAndLoadFile(f: File): void {
   const reader = new FileReader();
   reader.onload = () => {
     graph.load(reader.result as string);
   };
-
   reader.readAsText(f);
 }
 
-document.getElementById('bpmn-file').addEventListener('change', handleFileSelect, false);
+// TODO: move to UI initializer
+if (INITIALIZE_DRAG_AND_DROP) {
+  new DropFileUserInterface(window, 'drop-container', readAndLoadFile);
+} else {
```

Comment:
❓ why not keeping both the button in addition to the drag&drop feature
It is quite disturbing to see a blank page without knowning what to do.
We may add a message close to the button to show drag&drop support